.github/header-checker-lint.yml

```diff
@@ -0,0 +1,3 @@
+allowedCopyrightHolders:
+- 'Google LLC'
+- 'The Grafeas Authors'
```

Comment:
Add allowed copyright holders, the header check passed.

Reply:
Is this a header check bot configuration?

Reply:
Yes, please see https://github.com/googleapis/repo-automation-bots/tree/main/packages/header-checker-lint#configuration